### DISCUSSION TITLE
doubts regarding submission

### DISCUSSION THREAD

**Discussion by IllierZer** [larics/icuas22_competition#61] [Q&A]
@fpetric @lmark1  we are trying to run the script through the session.yml thing, initially we just launched the docker_run.sh and manually lauched our scripts from different terminals outside the docker environment. Now that in the solutions window in the session.yml we added the lines to launch the scripts from here. But the issue is that nothing is happening as in inside the docker environment we cannot access the code because as of now we have kept our scripts in the scripts directory inside the icuas22_competition but inside the icuas22_competition/scripts directory of the docker environment  our code is not there (only spawn_ball_at_uav.py is there). So can you please tell us how exactly to sort this issue, also apart from this can we modify the docker_run.sh file so as to mount the directory inside the docker environment and then access it from within the docker?

**Comment by lmark1** [comment]
Hello @IllierZer,

After modifying the repository in your local environment, you have to rebuild the Docker image in order for the changes to be copied in the image.

Best,
Lovro
